root/static/scripts/common/components/EntityLink.js

```diff
@@ -74,13 +74,23 @@ const iconClassPicker = {
 };
 
 const Comment = ({
+  alias,
   className,
   comment,
-}: {+className: string, +comment: string}) => (
+}: {+alias?: string, +className: string, +comment: string}) => (
   <>
     {' '}
     <span className={className}>
-      {bracketed(<bdi key="comment">{comment}</bdi>)}
+      {bracketed(
+        <bdi key="comment">
+          {nonEmpty(alias) ? (
+            <i title={l('Primary alias in your language')}>
```

Comment:
Should I change this now that it's not necessarily the user's language (due to falling back to English)?

Reply:
@reosarevok any suggestions? If we knew whether the alias was in English, I guess we could conditionally display "Primary alias in English" instead.

Reply:
Maybe "Primary alias" would be good enough here. I think the most important bit is to let editors know where this came from in case they want to change it.

Reply:
@Aerozol too. I've shortened it to "Primary alias" for now, but it'd probably be possible to also pass through and display the alias's locale as well (at the cost of a bit more code).

Reply:
Probably better to shorten it then, yeah - 'Primary alias' is still helpful in that it lets people know where the text comes from, and making the rollover too long seems annoying.
Some users will just have to figure out why it's sometimes in English and sometimes in their language, but that's not the most confusing thing in MB tbh :D

Reply:
P.S. But if anyone can figure out how to say this in 5 words or less, I will give them a big smooch: "Primary alias in your language or English as a fallback"

Reply:
I was thinking maybe "Localized primary alias", but "localized" is probably meaningless to many users. And we'd need to argue about whether to spell it "localized" or "localised". :-)